pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -73,6 +73,7 @@ let
 
       allNativeMessagingHosts = builtins.map lib.getBin (
         nativeMessagingHosts
+          ++ deprecatedNativeMessagingHost "enableBeidconnect" beidconnect
```

Comment:
These are to replace settings that were deprecated. We shouldn’t add more. You’re meant to set `nativeMessagingHosts` directly.